Initial introduction and getting started guides for ImageSharp.Drawing and Fonts (#12)
SixLabors/docs: articles/imagesharp.drawing/gettingstarted.md

# Getting Started

>[!WARNING]
>ImageSharp.Drawing is still considered BETA quality and we still reserve the rights to change the API shapes. WE are yet to priorities performance in our drawing APIs.

>[!NOTE]
>The official guide assumes intermediate level knowledge of C# and .NET. If you are totally new to .NET development, it might not be the best idea to jump right into a framework as your first step - grasp the basics then come back. Prior experience with other languages and frameworks helps, but is not required.

### ImageSharp.Drawing - Paths and Polygons

ImageSharp.Drawing provides several classes for build and manipulating various shapes and paths.

- @"SixLabors.ImageSharp.Drawing.IPath" Root interface defining a path/polygon and the type that the rasterizer uses to generate pixel output.
- This `SixLabors.ImageSharp.Drawing` namespace contains a variety of available polygons to speed up your drawing process.

> In addition to the vector manipulation APIs we also have the rasterization APIs that can convert your @"SixLabors.ImageSharp.Drawing.IPath"s to pixels.

Reviewer: ```suggestion
In addition to the vector manipulation APIs the library also contains rasterization APIs that can convert your @"SixLabors.ImageSharp.Drawing.IPath"s to pixels.
```

### Drawing Polygons

> ImageSharp provides several options for drawing polygons weather you want to draw outlines or fill the shapes.

Reviewer: ```suggestion
ImageSharp provides several options for drawing polygons whether you want to draw outlines or fill shapes.
```
